docs/src/modules/components/prism-okaidia.css

```diff
@@ -145,3 +145,8 @@ pre[class*="language-"] {
 .token.deleted {
   color: rgb(252, 146, 158);
 }
+
+.token.boolean,
+.token.number {
+  color: #b78eff;
```

Comment:
Reach AA contrast level. The problem is more visible since the introduction of code previews.